content/data/unicorns.json

```
@@ -230,5 +230,21 @@
 		"profileImg": "./maisydino.jpg",
 		"color": "#FDF6E3",
 		"roles": ["author"]
+	},
+	{
+		"id": "bobrossrtx",
+		"name": "Bobrossrtx",
+		"firstName": "Owen",
+		"lastName": "Boreham",
+		"description": "I have over 1000 years of software development experience, do not underestimate me!",
+		"socials": {
+			"twitter": "bobrossrtx",
+			"github": "bobrossrtx",
+			"website": "https://www.owenboreham.tech"
+		},
+		"pronouns": "he",
+		"profileImg": "./bobrossrtx.jpg",
+		"color": "#b7e11e",
+		"roles": ["developer"]
```

We need to add `author` to this, since you're adding an article! :) 

```suggestion
		"roles": ["developer", "author"]
```